spec/yaml/swiftnav/sbp/observation.yaml

```diff
@@ -209,7 +415,7 @@ definitions:
           desc: Mean motion difference
       - m0:
           type: double
-          units: radians
+          units: rad
```

Comment:
is it necessary to change this to `rad` from `radians`? is `rad` preferable in some way?